docs/tutorial/from-zero-to-hero-write-your-first-kubernetes-charm/index.md

```diff
@@ -0,0 +1,89 @@
+(from-zero-to-hero-write-your-first-kubernetes-charm)=
```

Comment:
I wonder if it would make it easier to work on things if the files in this section were numbered. It makes it more tricky to insert/remove chapters, but it would be more consistent with the repo and easier to see at a glance the structure.

Reply:
:+1: Tracked this in my actions doc